### PR TITLE
Surface Forward Headers in ext auth provider topics

### LINKED ISSUE #9549: Facebook redirectUri required https
Have integrated facebook external login but https is not enforced in my app in production, it is handled by load balancer which maintains certificate etc.  Is there a way for me to edit the redirect Uri sent to facebook to reference https instead of http?  Or am I doing it all wrong?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d1612f19-f267-493a-c230-2ca18e4eb50b
* Version Independent ID: d312db0f-6cf1-87ca-09b0-604a66e18a07
* Content: [Facebook, Google, and external provider authentication in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/?view=aspnetcore-2.1)
* Content Source: [aspnetcore/security/authentication/social/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/authentication/social/index.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #9549 

* Not a full UE pass on the group of external provider topics ... don't quite have time for that right now. However, I do hit a number of small UE-type updates.
* Engineering: You only need to look at one thing here: See the *forwarded-headers-middleware.md* file. The content of the file is below. The INCLUDE is linked into the external auth provider topics to surface Forwarded Headers Middleware (proxy/LB topic).

> \#\# Forward request information with a proxy or load balancer
>
> If the app is deployed behind a proxy server or load balancer, some of the original request information might be forwarded to the app in request headers. This information usually includes the secure request scheme (\`https\`), host, and client IP address. Apps don't automatically read these request headers to discover and use the original request information.
>
> The scheme is used in link generation that affects the authentication flow with external providers. Losing the secure scheme (\`https\`) results in the app generating incorrect insecure redirect URLs.
>
> Use Forwarded Headers Middleware to make the original request information available to the app for request processing.
>
> For more information, see \<xref:host-and-deploy/proxy-load-balancer>.

If it's easier to provide feedback in a comment here over hunting down the file on the diff, that works for me.

### CODE REVIEW

**Comment by guardrex** [comment]
@jkotalik Do you have a minute to look at this? It's just the content that I pasted into the opening comment :point_up:. The rest of the PR we'll review internally.

**Comment by guardrex** [comment]
Sorry @Tratcher ... I thought you were OOF.

**Comment by Tratcher** [comment]
I was but I'm back.

File: aspnetcore/security/authentication/social/index.md [hunk 1]
```diff
@@ -22,9 +23,9 @@ Note: Packages presented here abstract a great deal of complexity of the OAuth a
 
 ## Create a New ASP.NET Core Project
 
-* In Visual Studio 2017, create a new project from the Start Page, or via **File > New > Project**.
+* In Visual Studio 2017, create a new project from the Start Page, or via **File** > **New** > **Project**.
 
-* Select the **ASP.NET Core Web Application** template available in **Visual C# > .NET Core** category:
+* Select the **ASP.NET Core Web Application** template available in **Visual C#** > **.NET Core** category:
```
Comment (scottaddie) [inline aspnetcore/security/authentication/social/index.md]:
```suggestion
* Select the **ASP.NET Core Web Application** template available in the **Visual C#** > **.NET Core** category:
```

**Comment by guardrex** [comment]
The suggested change feature is 🎸!

**Pull request merged**